### PR TITLE
[OSSM-6569] Update images in CSV annotation to latest version

### PR DESCRIPTION
**Pull request by asmigala**

### CODE REVIEW

File: ossm/merge_upstream.sh [hunk 1]
```diff
@@ -49,13 +52,22 @@ merge() {
...
+    minor_version_underscore=${minor_version//./_}
+    sed -i -e "s/${minor_version}\.[0-9]\+/${latest_version}/g" -e "s/${minor_version_underscore}_[0-9]\+/${latest_version_underscore}/g" "${HELM_VALUES_FILE}"
+}
+
+main () {
   if ! merge; then
     set -e
     echo "Conflicts detected, attempting to run 'make gen' to resolve."
     rm -rf bundle/**/*.yaml resources bundle.Dockerfile
+    updateVersionsInOssmValuesYaml
```
Comment (FilipB) [inline ossm/merge_upstream.sh:68]: Minor suggestion. This needs to be run only when versions.yaml file is changed in upstream.

File: ossm/merge_upstream.sh [hunk 2]
```diff
@@ -49,13 +52,22 @@ merge() {
   return $?
 }
 
-main () {  
+updateVersionsInOssmValuesYaml() {
+    latest_version=$(yq '.versions[0].version' < "${VERSIONS_YAML_FILE}")
```
Comment (FilipB) [inline ossm/merge_upstream.sh:56]: Please double check yq is available on the host where this is running.

Comment (asmigala) [reply ossm/merge_upstream.sh:56]: I _believe_ this is run on `ci/maistra-builder:upstream-master` on the prow cluster, I pulled that image locally and verified that `yq` is there.

File: ossm/merge_upstream.sh (hunk 1 above)
Comment (FilipB) [inline ossm/merge_upstream.sh:68]: Also we don't want to run this for new minor versions, only for new micro versions

Comment (asmigala) [reply ossm/merge_upstream.sh:68]: The function is idempotent - if `versions.yaml` doesn't change, the result will stay the same.

**Review by FilipB — APPROVED** [review]
LGTM

**Pull request merged**